### PR TITLE
[DSM-2018] Add organizer information, date and venue TBD

### PR DESCRIPTION
**Pull request by espoelstra**

### CODE REVIEW

File: data/events/2018-des-moines.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Social Media Contact"
```
Comment (MikeRosTX) [inline data/events/2018-des-moines.yml]: Remove this Social Media Contact.

File: data/events/2018-des-moines.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    twitter: "devopsdaysdsm"
+  - name: "Tim Gifford"
+    github: "timgifford"
+    linkedin: "https://www.linkedin.com/in/timgifford"
+    twitter: "timgifford"
+  - name: "Jodi Jones"
+    github: "jodimj"
+    linkedin: "https://linkedin.com/in/jodi-jones-4a50ab17"
+    twitter: "jodimj"
+  - name: "Josh McCall"
```
Comment (MikeRosTX) [inline data/events/2018-des-moines.yml]: Send Josh McCall's email address to info@devopsdays.org so we can add him to the team email and to Slack before we can merge this PR.

Comment (bridgetkromhout) [inline data/events/2018-des-moines.yml]: I found his email in the earlier thread, so that's okay now.

File: data/events/2018-des-moines.yml (hunk 1 above)
Comment (bridgetkromhout) [inline data/events/2018-des-moines.yml]: Right, this shouldn't be here. This is meant to be specific humans only.

File: data/events/2018-des-moines.yml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  - name: "Jessie Puls"
+    github: "jessiepuls"
+    linkedin: "https://www.linkedin.com/in/jessiepuls"
+    twitter: "jessiepuls"
+  - name: "Ethan Spoelstra"
+    github: "espoelstra"
+    linkedin: "https://www.linkedin.com/in/ethanspoelstra"
+    twitter: "ewspoelstra"
+organizer_email: "devopsdaysdesmoines+organizers@gmail.com" # Put your organizer email address here
+proposal_email: "devopsdaysdesmoines+proposals@gmail.com" # Put your proposal email address here
```
Comment (bridgetkromhout) [inline data/events/2018-des-moines.yml]: These need to be organizers-des-moines-2018@devopsdays.org and proposals-des-moines-2018@devopsdays.org - the core team needs to stay in the loop.

**Comment by espoelstra** [comment]
How long will this take to show up on the main site as a TBD?

**Comment by bridgetkromhout** [comment]
> How long will this take to show up on the main site as a TBD?

It's there: https://www.devopsdays.org/events/2018-des-moines/

screen shot 2017-07-31 at 5 06 37 pm

**Pull request merged**